### DISCUSSION TITLE
My-Wayland-Your-Wayland-Our-Wayland/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#52] [Announcements]
# My-Wayland-Your-Wayland-Our-Wayland/

I <3 Open Source
That should be obvious by now, right? I’ve been out here blogging about Open Source stuff for over a decade, and occasionally I still have time to actually write code.

https://www.supergoodcode.com/My-Wayland-Your-Wayland-Our-Wayland/

**Comment by ms178** [comment]
As someone who is trained in a very different formalized (and often times nonsensical) framework of dispute resolution, Open Source Software development is insightful to watch, e.g. how engineers handle their disagreements. I hope no GPUs get hurt in the process. ;)

**Reply by zmike** [reply]
Pray for me and all my little GPUs.

**Comment by rhjdvsgsgks** [comment]
is the OpenGL ray-tracing being proposed?

**Comment by rhysperry111** [comment]
I hate it when Rhys steals my thunder

**Comment by wertig0n** [comment]
That picture is inaccurate. The dogs should be replaced by cats, and there should be at least a dozen more!

Other than that, that is indeed what peak Open Source performance looks like :)
